### ISSUE TITLE
Generate heroes randomly but based on specific parameters

### ISSUE THREAD

**Issue by elyukai** [elyukai/optolith-client#618]

**Comment by elyukai** [comment]
Related: https://www.reddit.com/r/DSA_RPG/comments/ao8lhq/odatas_meistertool_v021_gehen_open_source/

**Comment by madmath74** [comment]
Vielleicht kann man mit Meisterpersonen beginnen, die keine Talente, Vor- und Nachteile haben und kein Inventar?
Das würde die Generierung vereinfachen.
Wenn man dann den Erfahrungsgrad dieser NSC wählt, bekommt man ja die einzusetzenden AP.
Der Erstellungsworkflow wird ja vom Hauptprogramm vorgegeben (Rasse, Profession, Grundwerte, Ggfs Zauber), wobei für die weggefallenen Talente ja ein fester % Wert der AP abgezogen werden könnte.

Just my 2 cents :)

Grüße Micha

**Comment by elyukai** [comment]
Stimmt, dann hätte man Spezies, Kultur, Profession, Eigenschaften. Dann müste man noch schauen, auf welches Basis man die Attribute verteilt – also irgendwie muss ja klar sein, welche Eigenschaften für welche Profession sinnvoll(er) sind ... 🤔
